ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/AbstractBlockPropagationManagerTest.java

```diff
@@ -842,25 +845,27 @@ public void shouldThrowErrorWhenNoValidPeerAvailable() {
   }
 
   @Test
-  public void shouldStopWhenTTDReached() {
+  public void shouldStopWhenFinalized() {
```

Comment:
non-blocking, but an additional test that asserts block propagation never starts if we are past TTD would be a good measure.  I don't see a DefaultSynchronizerTest class however.